tests/commands/close.js

```diff
@@ -84,105 +60,112 @@ describe( 'commands/close', () => {
 						throw new Error( 'Supposed to be rejected.' );
 					},
 					response => {
-						expect( response.logs.error[ 0 ].split( '\n' )[ 0 ] ).to.equal( `Error: ${ error.message }` );
+						expect( response.logs.error[ 0 ].split( '\n' )[ 0 ] ).toEqual( `Error: ${ error.message }` );
 					}
 				);
 		} );
 
 		it( 'merges specified branch and remove it from local and remote', () => {
 			commandData.arguments.push( 'develop' );
 
-			stubs.execCommand.execute.onCall( 0 ).resolves( {
-				logs: {
-					info: [
-						'* develop'
-					],
-					error: []
-				}
-			} );
+			let execCall = 0;
+			execCommand.execute.mockImplementation( () => {
```

Comment:
Let's use mockResolvedValueOnce() (like in tests/commands/checkout.js) in all files, where we define custom output per function call.